recipes/pycld2/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "pycld2" %}
+{% set version = "0.31" %}
+{% set sha256 = "7d0f51b1550a6f964409f55f02b1f784d59a0d1b2718efa87c14d43344078403" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+  skip: true  # [win64]
+
+requirements:
+  build:
+    - python
+    - pip
+    - toolchain
+  run:
+    - python
+
+test:
+  imports:
+    - pycld2
+
+about:
+  home: https://github.com/aboSamoor/pycld2
+  license: APACHE
+  license_family: APACHE
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
```

Comment:
Thanks for packaging the missing license. Please make sure to open an issue upstream since Apache-2.0 requires that the license be distributed together with the source code, including the tar.gz file in pypi. 

If you want to go further, you can submit a PR similar to here kislyuk/yq#25.

Reply:
I will open this issue. Thank you for the reference!